pom.xml

```diff
@@ -289,6 +289,18 @@
                 <version>${mockito.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bc-fips</artifactId>
+                <version>1.0.2.4</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bctls-fips</artifactId>
+                <version>1.0.13</version>
```

Comment:
we should use the latest version, which is 1.0.17

Reply:
This is for version compatibility with ce-kafka

Reply:
that's weird, AFAIK, bctls-fips should be on version 1.0.17 for fips 140-2 https://confluentinc.atlassian.net/wiki/spaces/PM/pages/941887331/FedRAMP+Overview+Encryption+FIPS+Validation#Java